app/uk/gov/hmrc/cgtpropertydisposals/service/returns/ReturnsService.scala

```diff
@@ -61,7 +61,7 @@ import scala.util.Try
 @ImplementedBy(classOf[DefaultReturnsService])
 trait ReturnsService {
 
-  def submitReturn(returnRequest: SubmitReturnRequest)(
+  def submitReturn(returnRequest: SubmitReturnRequest, representeeDetails: Option[RepresenteeDetails])(
     implicit hc: HeaderCarrier,
```

Comment:
I'm just thinking is there anyway to avoid extending this method with an additional parameter - I would like to keep this method strictly a one parameter method that accepts a `returnRequest`. Extending it opens the doors to extending it ad-hoc in the future. Is there a way to extract the id and then update SubmitReturnRequest?

Reply:
I thought about updating copying the representee answers with the new cgt ref, but then that doesn't avoid the scenario you have to handle where you have to handle a `NoReferenceId` when trying to translate. We know we don't have to handle that case at that point because we should always pass in a valid cgt reference in that scenario

Reply:
hmm..something doesn't seem right. To me, when we reach `submitReturn` all information should have been captured already.